pkgs/top-level/all-packages.nix

```
@@ -3570,6 +3570,8 @@ with pkgs;
     electron = electron_11;
   };
 
+  coder = callPackage ../applications/misc/coder {};
```

Overrides should go here:
```suggestion
  coder = callPackage ../applications/misc/coder {
     ... = ...;
  };
```

---

I will remove the overrides :)